website/integrations/services/apple/index.md

```diff
@@ -0,0 +1,295 @@
+
+:::tip{title="OpenID Configuration URL"}
+
+Before leaving the provider overview page, make sure to copy the **OpenID Configuration URL** value to your text editor. You'll need this when configuring Apple Business Manager.
+
+You can always find your provider's generated URLs by navigating to **Providers**, then selecting the provider by name. The URLs are listed under the **Overview** tab.
+
+:::
+
+### Shared Signals Framework Provider
+
+While the OAuth2/OpenID Provider handles the authentication flow, you'll need to create a Shared Signals Framework Provider to handle the backchannel communication between authentik and Apple Business Manager.
+
+Once more from the authentik Admin interface, navigate to **Applications** -> **Providers** and press **Create**. This time, select **Shared Signals Framework Provider** and continue with the following form values:
+
+| Field | Value                      |
+| ----- | -------------------------- |
+| Name  | apple-business-manager-ssf |
+
+Press **Finish** to create the Shared Signals Framework Provider.
+
+### Application Configuration
+
+You're now ready to create the application configuration for Apple Business Manager. From the authentik Admin interface, navigate to **Applications** -> **Applications** and press **Create**.
+
+| Field                | Value                        |
+| -------------------- | ---------------------------- |
+| Name                 | Apple Business Manager       |
+| Slug                 | `abm`                        |
+| Provider             | Apple Business Manager       |
+| Backchannel Provider | `apple-business-manager-ssf` |
+
+With the application created, you're almost ready to switch over to Apple Business Manager.
+
+:::tip{title="SSF Config URL"}
+
+Assigning an SSF provider to a application will generate _SSF Config URL_ value. You'll need this when configuring Apple Business Manager, so make sure navigate to **Providers** -> **Apple Business Manager SSF** and copy the _SSF Config URL_ value to your text editor.
+
+:::
+
+---
+
+## Apple Business Manager Configuration
+
+Let's recap what you've done to configured authentik to act as an identity provider for Apple Business Manager.
+
+1. You've created three new scope mappings to handle the user's profile information, read access, and management access.
+2. Created a OAuth2/OpenID Provider to handle the authentication flow.
+    - Added the new scope mappings to the provider.
+    - And noted the _Client ID_, _Client Secret_, and _OpenID Configuration URL_.
+3. Created a Shared Signals Framework Provider to handle the backchannel communication between authentik and Apple Business Manager.
+    - Noted the _SSF Config URL_.
+4. Created application configuration for Apple Business Manager.
```

Comment:
This is of course accurate, but how can we word it to say that we "created a new application in authentik for ABM". This is the way our docs and UI present this... it's not so much "configuring an application" but rather creating, and then configuring an application.